tika/src/main/java/fr/pilato/elasticsearch/crawler/fs/tika/TikaInstance.java

```diff
@@ -94,11 +105,62 @@ private static void initParser(Fs fs) {
                         Collections.singletonList(TesseractOCRParser.class));
             }
 
-            Parser PARSERS[] = new Parser[2];
-            PARSERS[0] = defaultParser;
-            PARSERS[1] = pdfParser;
 
-            parser = new AutoDetectParser(PARSERS);
+            // load custom parsers if defined in config
+            if (fs.getCustomTikaParsers().size() > 0) {
+                Integer counter = 0;
+                Parser PARSERS[] = new Parser[fs.getCustomTikaParsers().size()+2];
+
+
```

Comment:
Can you remove some blank lines here? One is enough.

Reply:
Sure. Was not intentional.